File: python/examples/example_portcalls_port.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.port_calls(interval=720, extradata='voyage', locode='BGVAR'))
except ApiErrorException as e:
    print(e)

File: php/examples/example_portcalls_port.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $port_calls = $client->portCalls(720, ['locode' => 'BGVAR', 'extradata' => 'voyage']);
    print_r($port_calls);
    print_r($client->getLastInfo());
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}